google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -697,11 +697,6 @@ public BucketAccessControl getAcl(String bucket, String entity, Map<Option, ?> o
   }
 
   @Override
-  public BucketAccessControl getAcl(String bucket, String entity) {
```

Comment:
Shouldn't this method still exist and be provided as an overloaded method option?

Reply:
In the underlying diffbase PR #2558 , we decided to just add a parameter to the method signature instead of overloading it, even though this is a breaking change.

Reply:
I'd push not to cause breaking changes to the GA library and instead overloading.

Reply:
https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2558#discussion_r148646550
tl;dr overloading the method is a breaking change anyhow.

Reply:
IIUC, this only breaks code that overloaded the interface, but I'm not sure how often users overload the interface themselves versus not. I'd still vote to add overloaded methods.

Reply:
The StorageRpc (and this HttpStorageRpc) isn't really intended to be used by the general user. We'd assume they would use the Storage class, which wraps the StorageRpc calls in retry and exception-handling logic.

Reply:
Got it, thanks!